site/pages/view-all-disruptions.page.tsx

```diff
@@ -429,112 +516,105 @@ const ViewAllDisruptions = ({ disruptions, services, newDisruptionId }: ViewAllD
 
                     {filter.services.length > 0 ? <Table rows={formatServicesIntoRows(filter, setFilter)} /> : null}
 
-                    {startDateFilterError && (
+                    {(startDateFilterError || endDateFilterError) && (
```

Comment:
After the data is filtered based on start date and end date - if you delete the end date text then the error should be displayed but its not. 

If you remove both the start date and end date then the table contents should be reverted to the values before start date and end date were applied. This reset is not working.

Reply:
Testing this, if I enter a start and end date, and then start deleting the end date, I get an error saying an end date is needed.

Reply:
The issue happens when the user drags their cursor over the text and hits delete, as the form doesnt seem to register that the value has been deleted. I'm unsure what we want to do about this, because if we add an onBlur event to check if the input is empty, it will do that even if the user wasn't wanting to enter something.

Reply:
Fixed.

Reply:
Resolving this comment as a new bug will be raised for this issue